pkg/arvo/sys/zuse.hoon

```diff
@@ -5611,13 +5626,19 @@
       |*  {a/cord b/*}
       =>  .(+< [a b]=+<)
       [(rash a fel) b]
+    ::                                                  ::  ++pa:dejs:format
+    ++  pa                                              ::  string as path
+      (su ;~(pfix net (more net urs:ab)))
     ::                                                  ::  ++pe:dejs:format
     ++  pe                                              ::  prefix
       |*  {pre/* wit/fist}
       (cu |*(* [pre +<]) wit)
     ::                                                  ::  ++sa:dejs:format
     ++  sa                                              ::  string as tape
       |=(jon/json ?>(?=({$s *} jon) (trip p.jon)))
+    ::                                                  ::  ++se:dejs:format
+    ++  se                                              ::  string as @tas
```

Comment:
It seems a little strange to me to hard-code the aura here, as opposed to letting it be passed in as a parameter.  Something like this (I think it could stay dry).
```
++  se
  |=  aura=@tas
  |=  jon=json
  ?>  ?=([%s *] jon)
  (scot aura p.jon)
```

Reply:
Great idea. Thanks!

Reply:
Any reason why you wouldn't just specify it as

`
++  se
  |=  [aura=@tas jon=json]
  ?>  ?=([%s *] jon)
  (slav aura p.jon)
`
?

Reply:
Formatting was messed up, but the idea is why the second gate?

Reply:
Nevermind, Joe explained its use in function combinators such as cu.

Reply:
Completed this.